### PR TITLE
[SUBS-907] Apply Sample's new releaseDate field

### PR DESCRIPTION
**Pull request by ke4**

### CODE REVIEW

**Review by davidrichardson — APPROVED** [review]

**Comment by davidrichardson** [comment]
👍

File: src/main/java/uk/ac/ebi/subs/validator/biosamples/BiosamplesValidator.java [hunk 1]
```diff
@@ -76,38 +75,16 @@ private SingleValidationResult validateSampleName(Sample sample) {
...
 
-
-        String releaseDate = releaseDates.get(0).getValue();
-        try {
-            DateTimeFormatter formatter = DateTimeFormatter.ISO_DATE_TIME;
-            LocalDateTime.parse(releaseDate, formatter);
-        } catch (Exception e) {
-            logger.debug("Invalid date format: " + releaseDate);
-            singleValidationResult.setValidationStatus(SingleValidationResultStatus.Error);
-            singleValidationResult.setMessage(DATE_WRONG_FORMAT + ": " + e.getMessage());
```
Comment (fpenim) [inline src/main/java/uk/ac/ebi/subs/validator/biosamples/BiosamplesValidator.java:108]: This has already been approved, but this section on validation of the `releaseDate` format shouldn't have been deleted. BioSamples accepts a specific date format that we need to validate before accepting it.

**Comment by davidrichardson** [comment]
Is the format relevant now? We've restricted what receive to be a LocalDate, so we should always be able to use that in the agent to fulfill what BioSamples need

**Comment by ke4** [comment]
I agree with Dave. I think it is much easier to from the user perspective to give a LocalDate and we could transfer it to whatever BioSamples need.

**Pull request merged**